MonoGame.Framework/Graphics/Texture2D.cs

```diff
@@ -93,6 +97,7 @@ protected Texture2D(GraphicsDevice graphicsDevice, int width, int height, bool m
             this.GraphicsDevice = graphicsDevice;
             this.width = width;
             this.height = height;
+            this._texelSize = new Vector2(1f / (float)width, 1f / (float)height);
```

Comment:
@KonajuGames @dellis1972 - Any thoughts on this?

Since all the texture types are pretty much immutable when it comes to the size and type of texture data...  I don't find it all that bad to keep something like `TexelSize` as a helper.  It isn't so different than the sort key we added to `Texture`:

https://github.com/nkast/MonoGame/blob/develop/MonoGame.Framework/Graphics/Texture.cs#L27

The only thing I would like to see fixed if we add this is checking of the input arguments for `Texture2D`.  As the code is now you can pass 0 and get a divide by zero in this new code.  I suspect that `Texture2D` actually validates the width and height as well as the other inputs.